src/ralph/defaults.py

```diff
@@ -101,3 +101,5 @@ def config(key, default_value):
 )
 HISTORY_FILE = Path(config("RALPH_HISTORY_FILE", APP_DIR / "history.json"))
 LOGGING_CONFIG = config("RALPH_LOGGING", DEFAULT_LOGGING_CONFIG)
+SENTRY_DSN = config("RALPH_SENTRY_DSN", None)
+ENVIRONMENT = config("RALPH_ENVIRONMENT", "DEVELOPMENT")
```

Comment:
Environment may be confusing (may refer to environment variables), maybe prefer "execution environment"?

```suggestion
EXECUTION_ENVIRONMENT = config("RALPH_EXECUTION_ENVIRONMENT", "development")
```